### ISSUE TITLE
Go.mod file doesn't match in 1.0.2

### ISSUE THREAD

**Issue by hugbubby** [kevinburke/ssh_config#29]
I use a package downstream from this library, and since the release within the last hour when I try to update my go modules with `go get -u` it errors:
```
go get: github.com/kevinburke/ssh_config@v0.0.0-20190725054713-01f96b0aa0cd updating to
        github.com/kevinburke/ssh_config@v1.0.2: parsing go.mod:
        module declares its path as: github.com/virtuald/ssh_config
                but was required as: github.com/kevinburke/ssh_config```

**Comment by kevinburke** [comment]
I pushed an incorrect tag to the remote. I just removed the tag. The tag you want is "master", "latest" or "1.1"

**Comment by hugbubby** [comment]
> I pushed an incorrect tag to the remote. I just removed the tag. The tag you want is "master", "latest" or "1.1"

`go get -u` still seems to be erroring. How do I get it to target that tag?

**Comment by kevinburke** [comment]
this should work I think?

```
$ GO111MODULE=on go get -u github.com/kevinburke/ssh_config@latest
go: downloading github.com/kevinburke/ssh_config v0.0.0-20210327205412-beb851b6a483
```

**Comment by hugbubby** [comment]
> this should work I think?
> 
> ```
> $ GO111MODULE=on go get -u github.com/kevinburke/ssh_config@latest
> go: downloading github.com/kevinburke/ssh_config v0.0.0-20210327205412-beb851b6a483
> ```

It's still trying to get the 1.0.2 version with this command, or go get -u with no object. I've tried upgrading to ssh_config@1.1 but it seems to the 1.0.2 is a higher version than whatever it grabs from that.

**Comment by kevinburke** [comment]
Right, of course, the bad version is already cached in the proxy, and I don't use the proxies because they are vulnerable to exactly this issue.

Now I need to switch the versioning scheme for my project (which matches the Go versioning scheme). This is so frustrating.

**Comment by kevinburke** [comment]
I removed a bad release from my repository but it still appears in the mirror, what should I do?
Whenever possible, the mirror aims to cache content in order to avoid breaking builds for people that depend on your package, so this bad release may still be available in the mirror even if it is not available at the origin. The same situation applies if you delete your entire repository. We suggest creating a new version and encouraging people to use that one instead.

**Comment by hugbubby** [comment]
;_; sorry dude. I got this problem from including the pulumi automation engine, I wonder how many other people are inadvertently blocked right now.

**Comment by kevinburke** [comment]
I've just pushed a new tag that should fix the issue ("v1.1.0"). I'm extremely annoyed that I need to do this in perpetuity now.

**Comment by virtuald** [comment]
Oh. I'll send you an email separately.
